src/views/shared/SecurityBeacon.js

```diff
@@ -28,14 +43,19 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     el.css('background-image', '');
     el.removeClass('new-user undefined-user');
     if (isNewUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('new-user');
       return;
     }
     if (isUndefinedUser) {
+      el.find('.auth-beacon-description').text('no security image');
       el.addClass('undefined-user');
       return;
     }
     if (isSecurityImage) {
+      // TODO: Newer versions of qtip will remove aria-describedby on their own when destroy() is called.
+      el.removeAttr('aria-describedby');
```

Comment:
Does it affect the screen reader?
I thought the aria-describedby is not working (screen reader ignores it). That's why we workaround it. If the `aria-describedby` works, we should use it instead.